### ISSUE TITLE
Is not working with with npm 5.5.1 and nx 0.7.0-beta.1

### ISSUE THREAD

**Issue by tfiwm** [nrwl/nx#221]
Just for somebody who has the same error:

```javascript
TypeError: Invalid Version: file:.angular_cli165.tgz
    at new SemVer (/Users/mtschimev/workspace/porsche/pcc-applications/apps/my-porsche/node_modules/semver/semver.js:305:11)
    at /Users/mtschimev/workspace/porsche/pcc-applications/apps/my-porsche/node_modules/@angular/cli/bin/ng:146:29
    at /Users/mtschimev/workspace/porsche/pcc-applications/apps/my-porsche/node_modules/resolve/lib/async.js:45:21
    at ondir (/Users/mtschimev/workspace/porsche/pcc-applications/apps/my-porsche/node_modules/resolve/lib/async.js:196:27)
    at /Users/mtschimev/workspace/porsche/pcc-applications/apps/my-porsche/node_modules/resolve/lib/async.js:163:39
    at onex (/Users/mtschimev/workspace/porsche/pcc-applications/apps/my-porsche/node_modules/resolve/lib/async.js:104:32)
    at /Users/mtschimev/workspace/porsche/pcc-applications/apps/my-porsche/node_modules/resolve/lib/async.js:24:24
    at FSReqWrap.oncomplete (fs.js:153:5)
```

If you upgrade to npm 5.6.0 everything works 👍 

I recommend to use directly node 8.9.4 if you like LTS to have it directly on board ;)

**Comment by skydever** [comment]
thx for that :thumbsup:

**Comment by ThomasBurleson** [comment]
This has been fixed with release v0.8.0.
